components/tikv_util/Cargo.toml

```diff
@@ -45,7 +45,7 @@ sysinfo = "0.15"
 tikv_alloc = { path = "../tikv_alloc" }
 collections = { path = "../collections" }
 time = "0.1"
-tokio = { version = "0.2", features = ["rt-util"] }
+tokio = { version = "0.2", features = ["rt-util", "rt-core"] }
```

Comment:
Why rt-core is necessary?

Reply:
I introduced `tokio::task::LocalSet` several months ago